docs/migration/4_0.md

```diff
@@ -1,3 +1,8 @@
 # Technical Changes in Spartacus 4.0
 
 ## Breaking Changes Introduced in 4.0
+
+`ConfiguratorAttributeCheckboxListComponent` now also requires `ConfiguratorAttributeQuantityService`.
+`ConfiguratorAttributeDropDownComponent` now also requires `ConfiguratorAttributeQuantityService`.
+`ConfiguratorAttributeRadiButtonComponent` now also requires `ConfiguratorAttributeQuantityService`.
+`ConfiguratorCartEntryInfoComponent` now also requires `CommonConfiguratorUtilsService`.
```

Comment:
For every class, can we please:
- make the depdendency required
- drop deprecated constructor declarations?

FYI Sorry I didn't notice some of work might have beed already doubled in another ticket picked by the core team: https://github.com/SAP/spartacus/issues/10946

Reply:
But I recall a 3.4 release was still planned? Then we cannot do those changes now, right?